chain-network/proto/node.proto

```diff
@@ -61,31 +61,8 @@ message FragmentIds {
 // Request for peers
 message PeersRequest { uint32 limit = 1; }
 
-// Responses as a bunch of peers
-message PeersResponse { repeated Peer peers = 1; }
-
-// A Peer on the network
-message Peer {
-  // FIXME: make the port field common, indicate TCP protocol selection
-  // by the field number.
-  oneof peer {
-    PeerV4 v4 = 1;
-    PeerV6 v6 = 2;
-  }
-}
-
-// An ipv4 peer
-message PeerV4 {
-  fixed32 ip = 1;
-  fixed32 port = 2; // FIXME: uint32 is more efficient for port numbers
-}
-
-// An ipv6 peer
-message PeerV6 {
-  fixed64 ip_high = 1;
-  fixed64 ip_low = 2;
-  fixed32 port = 3; // FIXME: uint32 is more efficient for port numbers
-}
+// Responses as a bunch of peers, similar to Gossip
+message PeersResponse { repeated bytes peers = 2; }
```

Comment:
Is that to keep the backwards compatibility? Not sure if we care about it much right now.

Reply:
if you are talking about the field number yes, I used 2 so that it is (kind of) backward compatible.
 With regards to using a custom response message instead of reusing `Gossip`, I think it may be cleaner, and it is also what `TipResponse` does.